langfilter/langfilter.php

```diff
@@ -89,51 +111,71 @@ function langfilter_addon_settings_post(&$a,&$b) {
  *     of the user, then collapse the posting, but provide a link to
  *     expand it again.
  */
-function langfilter_prepare_body(&$a,&$b) {
 
-    $logged_user = local_user();
-    if ( ! $logged_user ) return;
+function langfilter_prepare_body(App $a, &$b)
+{
+	$logged_user = local_user();
+	if (!$logged_user) {
+		return;
+	}
 
-    # Never filter own messages
-    # TODO: find a better way to extract this
-    $logged_user_profile = $a->config['system']['url'] . '/profile/' . $a->user['nickname'];
-    if ( $logged_user_profile == $b['item']['author-link'] ) return;
+	// Never filter own messages
+	// TODO: find a better way to extract this
+	$logged_user_profile = $a->config['system']['url'] . '/profile/' . $a->user['nickname'];
+	if ($logged_user_profile == $b['item']['author-link']) {
+		return;
+	}
 
-    # Don't filter if language filter is disabled
-    if( get_pconfig($logged_user,'langfilter','disable') ) return;
+	// Don't filter if language filter is disabled
+	if (get_pconfig($logged_user, 'langfilter', 'disable')) {
+		return;
+	}
 
-    # Don't filter if body lenght is below minimum
-    $minlen = get_pconfig(local_user(),'langfilter','minlength');
-    if ( ! $minlen ) $minlen = 32;
-    if ( strlen($b['item']['body']) < $minlen ) return;
+	// Don't filter if body lenght is below minimum
+	$minlen = get_pconfig(local_user(), 'langfilter', 'minlength');
+	if (!$minlen) {
+		$minlen = 32;
+	}
+	if (strlen($b['item']['body']) < $minlen) {
+		return;
+	}
 
-    $spoken_config = get_pconfig(local_user(),'langfilter','languages');
-    $minconfidence = get_pconfig(local_user(),'langfilter','minconfidence');
+	$spoken_config = get_pconfig(local_user(), 'langfilter', 'languages');
+	$minconfidence = get_pconfig(local_user(), 'langfilter', 'minconfidence');
```

Comment:
Please replace the config functions with the new ones.

Reply:
Will do, thanks for the review.